### ISSUE TITLE
Time error on daemon

### ISSUE THREAD

**Issue by HazielKais** [pterodactyl/panel#1318]
**Background (please complete the following information):**
* Panel or Daemon: Daemon
* Version of Panel/Daemon: panel  0.7.10 / daemon 0.6.5
* Server's OS: CentOS
* Your Computer's OS & Browser: Deepin/Opera

**Describe the bug**
The time with deamon is incorrect but the time of the server is good ass seen here

> [root@webhost daemon]# date
>sam. sept.  8 13:47:06 CEST 2018
>[root@webhost daemon]# npm start

> pteronode@0.6.5 start /srv/daemon
> node src/index.js | node_modules/bunyan/bin/bunyan -o short

>11:47:10.591Z  INFO wings: + ------------------------------------ +
>11:47:10.594Z  INFO wings: | Running Pterodactyl Daemon v0.6.5    |
>11:47:10.595Z  INFO wings: |        https://pterodactyl.io        |
>11:47:10.595Z  INFO wings: |  Copyright 2015 - 2018 Dane Everitt  |
>11:47:10.595Z  INFO wings: + ------------------------------------ +

Panel: `PHP 7.2.9 (cli) (built: Aug 15 2018 08:05:45) ( NTS )` .
Daemon: `Linux webhost.CENSORED.fr 3.10.0-862.11.6.el7.x86_64 #1 SMP Tue Aug 14 21:49:04 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
` and `https://pastebin.com/AMAYHNzt` (your kernel version and information regarding docker)

**Comment by HazielKaos** [comment]
So, how can i change UTC to CEST ?

**Comment by parkervcp** [comment]
Easiest way is to check what your system time is set to - `timedatectl`

**Comment by HazielKaos** [comment]
Yep thx but how to change UTC to CEST in daemon cause console is not in the good date

**Comment by parkervcp** [comment]
Was your timezone in UTC when you ran that command?

**Comment by HazielKaos** [comment]
yup

**Comment by HazielKaos** [comment]
![image](https://user-images.githubusercontent.com/26751396/45254288-4205f680-b376-11e8-93c0-d2d238d79bbe.png)

**Comment by parkervcp** [comment]
You need to update your system timezone.
https://www.tecmint.com/check-linux-timezone/

That screenshot is useless.

**Comment by HazielKaos** [comment]
thx i ll try but for information the CPU graphic is in CEST 
![image](https://user-images.githubusercontent.com/26751396/45254306-914c2700-b376-11e8-9409-e64dded14b60.png)

**Comment by parkervcp** [comment]
Your CPU graph is generated on your system which is smart enough to convert from UTC to CEST.

**Comment by HazielKaos** [comment]
made this : `sudo ln  -sf /usr/share/zoneinfo/zoneinfo /etc/localtime`
but nothing change

![image](https://user-images.githubusercontent.com/26751396/45254340-f56eeb00-b376-11e8-804e-782d62c8ddbb.png)

**Comment by parkervcp** [comment]
That looks a lot you are running those in a game server... not in the server console.

**Comment by HazielKaos** [comment]
i have change /etc/locatime to Europe/Paris in server using ssh ;) and after i've send command from game server to see if anything change

**Comment by HazielKaos** [comment]
Ok so fixed it
need to add -Duser.timezone=Europe/SParis in launch args

**Comment by HazielKaos** [comment]
not realy beautifull but seem to work
![image](https://user-images.githubusercontent.com/26751396/45254406-cd33bc00-b377-11e8-9d89-c14668b43e98.png)
